storage/README.md

```diff
@@ -0,0 +1,4 @@
+collection of different storage interfaces that are shared by two or more components.
```

Comment:
this was my attempt to use READMEs (which are handy on github) to generate `doc.go` with md-to-godoc tool. But the tool has this difficulty with figuring out the first sentence.